atlp-rwanda/hackers-ec-be
# #187419055 Implemented user login with validation

**What does this PR do?**

This pull request implements the login feature which add the functionality for users to log into our E-commerce application using passport local strategy with their email and password

This involves creating the necessary backend endpoints, such as _/login_ and _/dashboard_ (for redirection)

**Endpoint for logging in:**

For now we have
`POST api/v1/users/login`

**How should this be manually tested?**

After cloning the repo and cd in it

-  add `.env` file and check for .env template in `.env.example `  ( I have added ACCESS_TOKEN_SECRET and SESSION_SECRET )
-  then, run `npm install`
-  After run `npm run dev` to start the server.
-  Use postman to test the login end-point provided above
-  Or test it using swagger on `http://localhost:[PORT]/api/v1/docs`

NOTE: -- _key_: Content-Type value: _application/json_

**Pivotal tracker story**

#187419055
